examples/dynamic/insurance_anthropic.py

```diff
@@ -88,22 +88,37 @@ class CoverageUpdateResult(FlowResult, InsuranceQuote):
 
 
 # Function handlers
-async def collect_age(args: FlowArgs) -> AgeCollectionResult:
-    """Process age collection."""
+async def collect_age(
+    args: FlowArgs, flow_manager: FlowManager
+) -> tuple[AgeCollectionResult, NodeConfig]:
+    """Process age collection and return next node name."""
     age = args["age"]
     logger.debug(f"collect_age handler executing with age: {age}")
-    return AgeCollectionResult(age=age)
 
+    flow_manager.state["age"] = age
```

Comment:
Went ahead and updated all the insurance examples to store age in `flow_manager.state` and read it back out in `collect_marital_status()`, which _seemed_ to be what was originally intended in this demo, but we had a limitation a while ago where function handlers didn't get `flow_manager`s passed to them.